docker/centos/misc

```diff
@@ -7,4 +7,4 @@ LABEL BUILD_ID=$LABEL_BUILD_ID
 
 COPY [".", "/etc/roaster/scripts"]
 
-RUN --mount=type=cache,id=ccache,target=/root/.ccache /etc/roaster/scripts/setup.sh catch2 jsoncpp rapidjson simdjson pybind grpc libpng libgdiplus mkl-dnn halide opencv leveldb rocksdb lmdb
+RUN --mount=type=cache,id=ccache,target=/root/.ccache /etc/roaster/scripts/setup.sh catch2 jsoncpp rapidjson simdjson pybind grpc libpng libgdiplus mkl-dnn halide opencv leveldb rocksdb lmdb utf8proc
```

Comment:
Recommend to put it right after simdjson as they're all "simple parsers".
Same for ubuntu.

Reply:
got it. will update it

Reply:
> Have you tested the approach on both Linux and Windows?

yes, I tested the nuget in win and lib in linux. Sorry that I didn't understand the scripts totally. I'll update both scripts according to your comments.